### PR TITLE
Jest/user controller

### PR DESCRIPTION
**Pull request by RudolphTorlage**
Added jest tests for the userController file.

### CODE REVIEW

File: tests-jest/userController.test.js
Comment (ChristopherPearse) [inline tests-jest/userController.test.js:1]: Added jest tests test the overall functionality of the controller, providing a good code coverage. Good job.

**Review by ChristopherPearse — APPROVED** [review]

File: tests-jest/userController.test.js [hunk 1]
```diff
@@ -68,3 +77,270 @@ describe('createUserAccount', () => {
...
+    const res = httpMocks.createResponse()
+
+    db.query.mockResolvedValueOnce({ rowCount: 0 })
+
+    await checkUserAccount(req, res)
+
+    expect(res._getRedirectUrl()).toBe('/login/?loginError=true')
+  })
+
+  it('should redirect to /landing if password matches', async () => {
```
Comment (samcohen1) [inline tests-jest/userController.test.js:101]: The tests for correct and incorrect logins have been implemented well.

File: tests-jest/userController.test.js [hunk 2]
```diff
@@ -68,3 +77,270 @@ describe('createUserAccount', () => {
...
+
+    await exitHistory(req, res)
+
+    expect(res._getRedirectUrl()).toBe('/')
+  })
+})
+
+// Additional tests for guest
+describe('guest', () => {
+  it('should set session user and redirect to /landing', async () => {
```
Comment (samcohen1) [inline tests-jest/userController.test.js:242]: Tests for Guest logins are also implemented well

**Review by samcohen1 — COMMENTED** [review]
The tests have been well implemented and good coding practices have been employed.

**Pull request merged**